components/engine/integration-cli/docker_cli_rmi_test.go

```diff
@@ -150,8 +150,8 @@ func (s *DockerSuite) TestRmiImageIDForceWithRunningContainersAndMultipleTags(c
 	dockerCmd(c, "tag", imgID, newTag)
 	runSleepingContainerInImage(c, imgID)
 
-	out, _, err := dockerCmdWithError("rmi", "-f", imgID)
-	// rmi -f should not delete image with running containers
+	out, _, err := dockerCmdWithError("rmi", imgID)
+	// rmi should not delete image with running containers
```

Comment:
If we're interested in preserving the intent  of this test I think the correct change is to check that this text is in stderr, but there is no error. It should continue to use `-f`